src/test/java/se/sundsvall/supportmanagement/integration/db/ErrandsRepositoryTest.java

```diff
@@ -127,6 +128,47 @@ void findByFilter(String filter) {
 				tuple("ERRAND_ID-1", "ASSIGNED_GROUP_ID-1", "ASSIGNED_USER_ID-1"));
 	}
 
+	@Test
+	void findByDateFilter() {
+		//Setup date filter
+		final var dateFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSXXX");
+		final var from = OffsetDateTime.now().minusMinutes(1).format(dateFormat);
+		final var to = OffsetDateTime.now().plusMinutes(1).format(dateFormat);
+		final var filter = String.format("(created > '" + from + "' AND created < '" + to + "')");
```

Comment:
This is the way String.format should be used:

`String.format("(created > '%s' AND created < '%s')", from, to);`